ca/certificate-authority.go

```diff
@@ -169,6 +190,14 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		NotAfter:   issuer.NotAfter,
 	}
 
+	// TODO(rlb@ipv.sx): Add a stop channel to shut down the CA gracefully
+	ca.signingQueue = make(chan signingRequest, signingQueueSize)
+	go func(src chan signingRequest) {
+		for req := range ca.signingQueue {
+			ca.sign(req)
```

Comment:
> What happens if the CA is kill -9'ed or crashes?

As this patch is right now, the remaining requests get dropped on the floor.  No different from what happens if the same thing happens right now.

The difference is in how this will look to the client.  In the status quo ante, the client would see the new-cert request fail or timeout, whichever happens to an RPC when an RPCServer dies mid-request.  With this patch, he'll see his certificate resource get stuck in the `pending` state.  We would probably want to periodically go through and move old pending certificates to `invalid`.  (TODO: Mark pending certificates with when they were added)